tensilelite/Tensile/Source/lib/include/Tensile/ContractionProblemPredicates.hpp

```diff
@@ -1015,6 +1015,83 @@ namespace Tensile
                 }
             };
 
+            struct AIGreaterThanEqual
+                : public Predicate_CRTP<AIGreaterThanEqual, ContractionProblem>
```

Comment:
References to ContractionProblem will become ContractionProblemGemm, see other note for more info.